debian/rules

```diff
@@ -22,6 +22,19 @@ KERNEL_TYPE.esx := generic
 KERNEL_TYPE.gcp := gcp
 KERNEL_TYPE.kvm := kvm
 
+#
+# The DELPHIX_SIGNATURE_VERSION field refers to the current Delphix
+# Release version being built and only defaults to 6.0 if not set.
+# This is passed as an environment variable via jenkins job(s) that
+# launch the linux-pkg build for this package.
+#
+DELPHIX_SIGNATURE_VERSION := $(or $(DELPHIX_SIGNATURE_VERSION),6.0)
```

Comment:
By the way, the name `DELPHIX_SIGNATURE_VERSION` might be a little confusing. It represents the version of the public key pair (upgrade, registration) to use hereon in the appliance. It's not necessarily the version of the signature included in the upgrade image where this package is bundled in.

Reply:
For now decided to keep the same name, unless you feel strongly otherwise about this, thanks.